CLI pages for review (#465)
newrelic/developer-website: src/markdown-pages/nr1-cli/nr1-subscription.mdx

---
path: '/explore-docs/nr1-subscription'
title: 'New Relic One CLI subscription commands'
template: 'GuideTemplate'
description: 'An overview of the CLI commands you can use to manage your Nerdpack subscriptions.'
tileShorthand:
  title: 'Subscription CLI commands'
  description: 'Manage your Nerdpack subscriptions with the New Relic One CLI.'
redirects:
---
<Intro>

> To manage your Nerdpack subscriptions, use the commands below. You can click any command to see its usage and options, among other details.

Reviewer: ```suggestion
To manage your Nerdpack subscriptions, use the commands below.  You can click any command to see it's usage options and additional details about the command.
```

Reviewer 2: s/b "its"
